### DISCUSSION TITLE
[A/L] (2021/22) Induzione - Esercizio 1

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#301] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152321569-7d428fd0-f8fb-42a7-b88c-345645f9f3f7.png)

**Comment by danieldinella** [comment]
Il caso base ovvero n = 12 si ottiene da 3 francobolli da 4. Da lì in poi, per incrementare di 1, sostituiamo un 4 con un 5. Infine quando ci troviamo in una situazione di soli 5 mettiamo tutti 4 e aggiungiamo un 4.

**Comment by Exyss** [comment]
Soluzione analoga a quella di @danieldinella 
![Esercizio 1](https://user-images.githubusercontent.com/65043207/152642969-cd90302f-d526-4591-956d-a34faaaee94c.jpg)

**Comment by LuginiAndrea** [comment]
Riformuliamo il problema:
$$\forall n \in \mathbf{N} | n \geq 12 \implies \exists j, k \in \mathbf{N} \textrm{ t.c. } n = 4j + 5k$$
Caso base: $n = 12 \implies n = 4 \times 3 \implies j = 3, k = 0$. 
Passo induttivo:
$$n + 1 = 4j + 5k + 1 \textrm{ per induzione } = 4j + \left(4 + 1\right)k + 1$$
$$= 4\left(j-1\right) + \left(4+1\right)k + \left(4 + 1\right) = 4\left(j-1\right) + \left(4+1\right)\left(k+1\right)$$
$$=  4\left(j-1\right) + \left(5\right)\left(k+1\right)$$
Questo è valido se $j \geq 1$
Nel caso $j = 0$ 
$$n + 1= 5k + 1 \textrm{ per induzione } = 5 \times 3 + 5\left(k-3\right) + 1 = 16 + 5\left(k - 3\right) = 4 \times 4 + 5\left(k-3\right)$$
Dimostriamo che $j = 0 \implies k \geq 3$ (ovvero che una delle due strategie è sempre usabile)
$j = 0$ implica che $n$ è un numero che si può scrivere come $5k$. Il più piccolo numero con tale proprietà e che è $\geq 12$ è $15 =5 \times 3$, di conseguenza qualsiasi altro numero nella forma $5k$ avrà $k > 3$

**Comment by ATLANCTIX4** [comment]
> Soluzione analoga a quella di @danieldinella ![Esercizio 1](https://user-images.githubusercontent.com/65043207/152642969-cd90302f-d526-4591-956d-a34faaaee94c.jpg)

il caso base non dovrebbe essere:
3X4cent + 0X5cent = 12?
perche hai scritto 4X4 cent?

**Comment by Duvan-Zapat** [comment]
Il professor 

> Soluzione analoga a quella di @danieldinella ![Esercizio 1](https://user-images.githubusercontent.com/65043207/152642969-cd90302f-d526-4591-956d-a34faaaee94c.jpg)

Il professore piperno nell'appello di giugno ha detto che la parte induttiva è sbagliata, come mai?
